src/components/global/SandboxPage/SandboxPage.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from "react";
+import { SandboxIframe } from "../../utility/SandboxIframe/SandboxIframe"; 
+
+interface SandboxPageProps {
+    title: string,
+    description: {
+        before: Element | ReactElement;
+        after: Element | ReactElement;
```

Comment:
```suggestion
        before: ReactNode;
        after: ReactNode;
```
Do you expect only a single `element` or you accept multiple elements?

Reply:
Single element, so your suggestion is correct 👍 Thanks!